docs/management/admin/policy-list.asciidoc

```diff
@@ -0,0 +1,10 @@
+[[policy-list]]
+[chapter]
+= Policy list
+
+The **Policy List** page lists all of the integration policies configured for {endpoint-sec}. 
+
+Click on an integration policy's name to view details and configure policy settings. For more information on configuring an integration policy, refer to <<configure-endpoint-integration-policy>>.
```

Comment:
```suggestion
Click on an integration policy's name to view its details and settings. For more information on configuring an integration policy, refer to <<configure-endpoint-integration-policy>>.
```
Just a suggestion. I think if you are going to be able to view settings its generally assumed you can change them.

Reply:
@benironside I agree, we don't need two verbs here. Actually I think "configure" is the action to keep here, since the user can't do that from the Policies page itself but might be expecting to.
```suggestion
Click on an integration policy's name to configure its settings. For more information on configuring an integration policy, refer to <<configure-endpoint-integration-policy>>.
```